### DISCUSSION TITLE
secure-someip integration

### DISCUSSION THREAD

**Discussion by lpasarel** [COVESA/vsomeip#637] [Q&A]
Hi vsomeip team

There's this proposal made by @giorio94 for secure-vsomeip
https://github.com/netgroup-polito/secure-vsomeip

Did anyone looked on it? 
Is this proposal something that can be considerd for the future?

regards

**Comment by goncaloalmeida** [comment]
hello @lzala 
yes is something that could be considered! we can discuss in our next monthly week this approach if needed.
cc @lutzbichler @fcmonteiro @BrunoLSilvaCTW
